stix_shifter/stix_transmission/src/modules/async_dummy/api_client.py

```diff
@@ -4,6 +4,15 @@
 class APIClient():
 
     def __init__(self, connection, configuration):
+        # auth = configuration.get('auth')
```

Comment:
can we put a comment here to specify the reason of commenting out this block of code to avoid confusion of "some" people? it would have been ok if it is a private repo.